pyproject.toml

```
@@ -0,0 +1,53 @@
+[build-system]
+requires = [
+    "setuptools>=77",
+    "wheel",
+    "setuptools-scm",
+]
+
+[project]
+name = "circuitpython-build-tools"
+requires-python = ">=3.10"
+description = "CircuitPython library build tools"
+readme = "README.md"
+authors = [
+    {name = "Scott Shawcroft", email = "scott@adafruit.com"}
+]
+keywords = [
+    "circuitpython",
+]
+license = "MIT"
+classifiers = [
+    # https://pypi.org/pypi?%3Aaction=list_classifiers
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Libraries",
+    "Programming Language :: Python :: 3",
+]
+dynamic = ["version", "dependencies"]
+
+[project.scripts]
+circuitpython-build-bundles = "circuitpython_build_tools.scripts.build_bundles:build_bundles"
+circuitpython-mpy-cross = "circuitpython_build_tools.scripts.circuitpython_mpy_cross:main"
+
+[project.urls]
+Source =  "https://github.com/adafruit/circuitpython-build-tools"
```

Missed this field from setup.py
```suggestion
Homepage = "https://www.adafruit.com/"
Source =  "https://github.com/adafruit/circuitpython-build-tools"
```